Homework3+4

dmitriymitr opened the pull request:
Hw 3
   debug 'debug.py'

Hw 4
   Change list.html with master.html
   todo_item make DB and change view, models, admin

dmitriymitr committed:
Homework3+4

Hw 3
   debug 'debug.py'

Hw 4
   Change list.html with master.html
   todo_item make DB and change view, models, admin

sergkot2020 commented on todo/todo_item/models.py:
```diff
@@ -1,3 +1,18 @@
+    created = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    user = models.ForeignKey('auth.User', on_delete=models.CASCADE)
+    list_model = models.CharField(max_length=256, verbose_name='Список')
```
Тут ошибка,
User поле не нужно, а поле 
List_model должно быть ForeignKey ('main.List'

dmitriymitr committed:
Homework3+4

Hw 3
   debug 'debug.py'

Hw 4
   Change list.html with master.html
   todo_item make DB and change view, models, admin
